tests/components/onewire/test_options_flow.py

```diff
@@ -0,0 +1,237 @@
+"""Tests for 1-Wire config flow."""
```

Comment:
These tests should move to the `test_config_flow.py` module.

Reply:
The reason they are separate is that they needed  different mock-setups. This may perhaps sort itself later, when the device-registry comment is handled. (But I can't say for sure )